src/mixins/I13nUtils.js

```diff
@@ -12,29 +13,33 @@ var I13nUtils = {
         i13n: React.PropTypes.shape({
             executeEvent: React.PropTypes.func,
             getI13nNode: React.PropTypes.func,
-            parentI13nNode: React.PropTypes.object
+            parentI13nNode: React.PropTypes.object,
+            reactI13nInstance: React.PropTypes.object
         })
     },
     
     childContextTypes: {
         i13n: React.PropTypes.shape({
             executeEvent: React.PropTypes.func,
             getI13nNode: React.PropTypes.func,
-            parentI13nNode: React.PropTypes.object
+            parentI13nNode: React.PropTypes.object,
+            reactI13nInstance: React.PropTypes.object
         })
     },
     
+    
     /**
      * getChildContext
      * @method getChildContext
      */
     getChildContext: function () {
         return {
-            i13n: {
+            i13n: Object.assign({}, this.context.i13n, {
                 executeEvent: this.executeI13nEvent,
                 getI13nNode: this.getI13nNode,
-                parentI13nNode: this._i13nNode
-            }
+                parentI13nNode: this._i13nNode,
+                reactI13nInstance: this._getReactI13n()
```

Comment:
this is internally usage, we use this context to pass reactI13n instance(and the i13n root)  across components, do we want users to know this?